app/src/main/java/com/dkanada/gramophone/ui/activities/SplashActivity.java

```diff
@@ -50,6 +43,12 @@ protected void onCreate(Bundle savedInstanceState) {
         login();
     }
 
+    @Override
+    public void onPause() {
+        super.onPause();
+        overridePendingTransition(0, 0);
+    }
+
```

Comment:
You might want to add this to the login activity as well because there's a pretty long transition from login to splash if I recall correctly. There's also the rare case of main to login when a user logs out that might have had issues.